docker-compose/promscale-demo/docker-compose.yaml

```diff
@@ -40,12 +40,9 @@ services:
       - ${PWD}/../rules.yml:/rules.yml
       - ${PWD}/../alerts.yml:/alerts.yml
     environment:
-      PROMSCALE_WEB_TELEMETRY_PATH: /metrics-text
-      PROMSCALE_DB_URI: postgres://postgres:password@timescaledb:5432/otel_demo?sslmode=allow
-      PROMSCALE_TELEMETRY_TRACE_OTEL_ENDPOINT: "collector:4317"
-      PROMSCALE_TELEMETRY_TRACE_SAMPLING_RATIO: "0.1"
+      PROMSCALE_DB_URI: postgres://postgres:password@timescaledb:5432/tsdb?sslmode=allow
       PROMSCALE_METRICS_RULES_CONFIG_FILE: /prometheus.yml
-      TOBS_PROMSCALE_QUICK_START: "true"
+      PROMSCALE_PKG: "docker-quick-start"
```

Comment:
What does this do?

Reply:
This will add metadata in our telemetry to identify that this deployment of Promscale is using the quick start